ui.go

```diff
@@ -296,28 +296,32 @@ func (win *win) printDir(dir *dir, marks map[string]int, saves map[string]bool)
 	end := min(beg+win.h, maxind+1)
 
 	for i, f := range dir.fi[beg:end] {
-		switch {
-		case f.linkState == working:
-			fg = termbox.ColorCyan
-			if f.Mode().IsDir() {
-				fg |= termbox.AttrBold
-			}
-		case f.linkState == broken:
-			fg = termbox.ColorMagenta
-		case f.Mode().IsRegular():
-			if f.Mode()&0111 != 0 {
-				fg = termbox.AttrBold | termbox.ColorGreen
-			} else {
-				fg = termbox.ColorDefault
+		if gOpts.lscolors {
+			fg, bg = lsColors.getColors(f)
+		} else {
```

Comment:
Can we use your new functions to remove all these former code in the else branch? These were written in a hurry and I doubt they are correct and complete. When there is no `LSCOLORS` or `LS_COLORS` variables set, we could use the values returned by `parseBSD` function with the value `"exfxcxdxbxegedabagacad"` as documented in FreeBSD and Darwin man pages. Maybe `lscolors.go` file should simply be `color.go` and all color logic should be moved there.